pytorch_lightning/callbacks/quantization.py

```diff
@@ -114,6 +118,11 @@ def __init__(
             observer_type: allows switching between ``MovingAverageMinMaxObserver`` as "average" (default)
                 and ``HistogramObserver`` as "histogram" which is more computationally expensive.
 
+            max_num_inputs: maximum number of inputs nodes used by the model in training, evaluation, or
```

Comment:
Can you be more explicit about what are inputs exactly?

Is it only batches? I see you used `len(tensor)` to get the first dimension.

Same applies to other mentions of "inputs" in this PR

Reply:
Do these latest two commits add enough clarity in this area?